### ISSUE TITLE
Module breaks after packaging the application with NPM PKG

### ISSUE THREAD

**Issue by mikehajj** [fastify/fastify-swagger#361]
## 🐛 Bug Report

This module breaks after packaging the application that is using it with [npm pkg](https://www.npmjs.com/package/pkg).
I am tasked to create a microservice using Fastify with some APIs and then package it into an executable using **PKG** module.
After building and testing the application, everything works, I added the fastify-swagger module to generate the documentation for the APIs. All works fine until I package, the module breaks but the APIs work. 

## To Reproduce

Download the zip file I attached and extract it, it contains 3 files: 

- package.json 
- index.js ( taken from the examples of this repo ) 
- build.sh the packaging script

```js
$ npm install -g pkg
$ cd pkg-test

# start the server as a node application
$ npm install 
$ node index.js

# create a package
$ ./build.sh

# start the package server
$ ./build/server
```

## Expected behavior

Once the application starts, in your browser navigate to HTTP://localhost:3000/documentation and it should open the swagger documentation.

However, if you package the application then start it, this link doesn't work and instead, you see the attached screenshot.
I have been battling with this for a couple of days now and still have no idea why this happens.

![Screen Shot 2021-02-09 at 10 24 40 AM](https://user-images.githubusercontent.com/1814917/107335835-147e8380-6ac1-11eb-9e70-2d327d356ebe.png)

## Environment

- *node version*: 14
- *fastify version*: >=3.8.0
- *os*: Mac & Linux (ubuntu)

## Copy of the Code

[pkg-test.zip](https://github.com/fastify/fastify-swagger/files/5949868/pkg-test.zip)

**Comment by mcollina** [comment]
Thanks for reporting this @mikehajj! This module does not really claim (or even test), that running it via `pkg` would work in any way. There is likely some incompatibility between the two.

We are happy to review any changes you think are needed to make this work here. Would you like to send a PR?

**Comment by mikehajj** [comment]
@mcollina thanks for the prompt reply. 
I am sorry I got excited and reported the issue. After I posted the bug, an idea popped into my head and that solved the problem.

The issue turned out to be in the configuration of PKG in my package.json and had nothing to do with the module.

in the package.json of my application and specifically in the configuration section of pkg, I needed to add the module as an asset and that solved the problem.

```js
"pkg": {
    "scripts": [
      "*.js"
    ],
    "assets": [
      "node_modules/fastify-swagger"
    ]
  }
```
After adding the entry in the assets section, pkg started taking the files in fastify-swagger into consideration and especially the files under the **static** folder ( the css, HTML and swagger-ui files ).

There is no need for PR, maybe a simple note in the documentation that if the module is used with PKG, make sure you add node_modules in the assets section so that things don't break.

Again thanks for the reply and sorry if I have wasted anyone's time.
